src/test/groovy/com/github/rodionovsasha/commentservice/services/CommentServiceTest.groovy

```diff
@@ -190,4 +190,14 @@ class CommentServiceTest extends BaseTest {
         then:
         repository.getOne(ARCHIVED_COMMENT_ID).archived
     }
+
+    def "findByTopic returns not archived comments for topic ASC sorted"() {
+        expect:
+        commentService.findByTopic(TOPIC_ID).id == [4, 5, 6, 8]
+    }
+
+    def "findByTopic returns empty list when topic not found"() {
```

Comment:
This might be wrong, probably, we need to check whether topic exists otherwise to throw an exception